src/containerapp/azext_containerapp/_utils.py

```diff
@@ -691,3 +691,16 @@ def log_in_file(log_text, opened_file, no_print=False):
 def remove_ansi_characters(text):
     regular_expression = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
     return regular_expression.sub("", text)
+
+
+# Remove null/None properties in a model since the PATCH API will delete those. Not needed once we move to the SDK
+def clean_null_values(d):
```

Comment:
why do we need this change

Reply:
Because the `clean_null_values` in import from azure-cli before. And the difference is:
Change from
```
if v or isinstance(v, list)
```
to
```
if v is not None or isinstance(v, list)
```
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/a887e122-fae3-4a7f-be8a-300f1b683747)

Reply:
After discussing with zunli, in this PR we fix the --min-replicas first. For --yaml, we fixed in the azure-cli.